### PR TITLE
csq

### PR DESCRIPTION
**Pull request by Sunlly0**
### Summary of talk

Date:

Topic:

Keywords:

### Format check List

- [ ] Conforming to English specifications especially in `README.md`.
- Capitalize the first letter.
- Use English punctuation with appropriate space.
- ...
- [ ] Remember to add links to reference papers.
- [ ] Make sure the link is accessible.
- Use dash `-` to connect filename rather than space ` `, the latter will casue dead link.
- [ ] Make sure latex formulas display normally, more can be found in [#31](https://github.com/CDDSCLab/Weekly-Group-Meeting-Paper-List/discussions/31).

### CODE REVIEW

File: meeting-summary/2022-09-09-表格开领域问答.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+### **BM25**
+
+BM25（BM：best matching）算法是一种计算句子与文档相关性的算法。先将输入的句子分词，然后计算句子中每个词与文档的相关性，最后进行加权求和，得到句子和文档的相关性评分。
+
+计算公式如下：
+
+![Untitled](imgs/image-20220909-9.png)
+
+- D: 文档， Q：问句
+- q_i：问句中的各个分词
```
Comment (ehds) [inline meeting-summary/2022-09-09-表格开领域问答.md:74]: Github has supported mathematical expressions in markdown, mathematical expression should be wrapped in `$` like that `$q_i$`, which will be render as $q_i$.

more discussion please refer to https://github.com/CDDSCLab/Weekly-Group-Meeting-Paper-List/discussions/31#discussioncomment-2852396

**Comment by IcePigZDB** [comment]
CI lint check has been fixed in #43, rebase the main branch and resolve conflicts to pass it~

**Review by ehds — APPROVED** [review]
LGTM

**Review by IcePigZDB — APPROVED** [review]
LGTM

**Pull request merged**